src/index.html

```diff
@@ -5,9 +5,38 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card">
+      <div class="card__block card__block--margin" data-qa="card">
+        <img src="./images/imac.jpeg" alt="imac"
+        class="photo">
+        <div class="container">
+          <h1 class="container__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+          <p class="container__code">Product code: 195434</p>
+          <div class="container__stars">
+            <div class="stars">
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+              <div class="star"></div>
+            </div>
+            <div class="reviews">
+              <h1 class="rev_text">Reviews: 5</h1>
+            </div>
+          </div>
+          <div class="container__price">
+            <span class="price-text">Price:</span>
+            <span class="price-cost">$2,199</span>
+          </div>
+          <div class="container__link container__link--margin">
+            <a class="link" href="#"
+            data-qa="hover">buy</a>
```

Comment:
code is not aligned, check the checklist 
```suggestion
            <a 
              class="link" 
              href="#"
              data-qa="hover"
            > buy </a>
```
<img width="1133" alt="Screenshot 2023-09-19 at 13 03 48" src="https://github.com/mate-academy/layout_product-cards/assets/64746943/14f46eef-24a3-44df-b020-a0a3c6a2bc48">